### DISCUSSION TITLE
How to make a which key keymap in Lazyvim?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1844] [Q&A] [answered]
I use lunarvim before which define which-key by 
`lvim.builtin.which_key.mappings["cw"] = {'<cmd>lua ChangeWallpaper()<CR>', "ChangeWallpaper", silent=true }`
I check the doc of lazyvim not find related manual. Is there any builtin method for which-key so I can use `lazyvim` way to make it?

**Comment by dpetka2001** [comment, marked as answer]
Plenty of examples in the docs [here](https://www.lazyvim.org/plugins/ui#which-keynvim), [here](https://www.lazyvim.org/plugins/editor#which-keynvim) and [here](https://www.lazyvim.org/extras/dap/core#which-keynvim-optional) and probably other places as well. Basically, you just have to do in a file in `/lua/plugins/` directory the following
```lua
{
  "folke/which-key.nvim",
  opts = {
    defaults = {
      ["<leader>d"] = { name = "+debug" },
    },
  },
}
```
Just put your own values that you would like. Although in your case, from the example code that you used, a simple keymap in `/lua/config/keymaps.lua` should do it as long as you provide a `desc` for the keybinding that you want, `which-key` should be able to pick up the description and show the keymap.
